tests/dag_processing/test_job_runner.py

```diff
@@ -762,6 +763,95 @@ def test_scan_stale_dags_standalone_mode(self):
             active_dag_count = session.query(func.count(DagModel.dag_id)).filter(DagModel.is_active).scalar()
             assert active_dag_count == 1
 
+    @conf_vars(
+        {
+            ("scheduler", "standalone_dag_processor"): "False",
```

Comment:
This is False by default? Should we check too that when enabled, it doesn't affect standalone processor

Reply:
We can remove the conf_vars then since this config is False by default

Reply:
Sure, I have removed it.